mass_mailing_custom_unsubscribe_event/demo/assets.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - João Marques
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<template id="assets_frontend_demo"
+          inherit_id="web.assets_frontend">
+    <xpath expr=".">
+        <script type="text/javascript"
+                src="/mass_mailing_custom_unsubscribe_event/static/src/js/tour.js"/>
+    </xpath>
+</template>
+
+<!--This is set here to make tours work-->
+<template id="mass_mailing_custom_unsubscribe.layout"
```

Comment:
:warning: Beware of this. You should *never* declare in a module a data for other module. Otherwise uninstalling one won't uninstall this asset.

```suggestion
<template id="layout"
```

Reply:
Thanks.
I got a little confused as I used the example from `mass_mailing_custom_unsubscribe` and also because when using it as you suggested, I got errors in the tests.
As we are inheriting from it, I ended up removing that part and the tests and tour work as usual.